python/lsst/daf/butler/_butler_config.py

```diff
@@ -191,3 +194,20 @@ def __init__(
         # Not needed if there is never information in a butler config file
         # not present in component configurations
         self.update(butlerConfig)
+
+    def get_butler_type(self) -> ButlerType:
+        # Configuration optionally includes a class name specifying which
+        # implementation to use, DirectButler or RemoteButler.
+        butler_class_name = self.get("cls")
+        if butler_class_name is None:
+            # There are many existing DirectButler configurations that are
+            # missing the ``cls`` property.
+            return ButlerType.DIRECT
+        elif butler_class_name == "lsst.daf.butler.direct_butler.DirectButler":
+            return ButlerType.DIRECT
+        elif butler_class_name == "lsst.daf.butler.remote_butler.RemoteButler":
+            return ButlerType.REMOTE
+        else:
+            raise ValueError(
+                f"Butler configuration requests to load unknown Butler class {butler_class_name}"
```

Comment:
```suggestion
                f"Butler configuration requests to load unknown Butler class {butler_class_name!r}"
```
so the class name is quoted.